.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] I noticed that only `node_module` is ignored in your gitignore file. Kindly add the `test.md` file to your root directory and inside the file write `This file should be ignored by Git` and the test.md file should be ignored the same way you have ignored the node_modules as requested in the project requirement. 👍 👍